# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/silo/tilo-v8.patch
# Copyright (C) 2020 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- silo-fcf8ffb4fba3ea72353c111ab7dc210eaae89223/tilo/Makefile.vanilla	2020-10-28 12:15:14.162698940 +0100
+++ silo-fcf8ffb4fba3ea72353c111ab7dc210eaae89223/tilo/Makefile	2020-10-28 12:16:18.686698176 +0100
@@ -5,5 +5,7 @@
 
 include ../Rules.make
 
+CFLAGS += -mcpu=v8
+
 # Relocate to 3.75MB
 # Hopefully, 4MB are guaranteed to be mapped.
